test/unit/mods.example.core.audioware.ts

```diff
@@ -0,0 +1,137 @@
+      expectedInstallerType: InstallerType.CoreAudioware,
+      inFiles: [
+        path.join(`red4ext\\`),
+        path.join(`red4ext\\plugins\\`),
+        path.join(`red4ext\\plugins\\Audioware\\`),
+        path.join(`red4ext\\plugins\\Audioware\\audioware.dll`),
+        path.join(`r6\\scripts\\Audioware\\Codeware.reds`),
+        path.join(`r6\\scripts\\Audioware\\Config.reds`),
+        path.join(`r6\\scripts\\Audioware\\Ext.reds`),
+        path.join(`r6\\scripts\\Audioware\\Hooks.reds`),
+        path.join(`r6\\scripts\\Audioware\\Natives.reds`),
+        path.join(`r6\\scripts\\Audioware\\Preset.reds`),
+        path.join(`r6\\scripts\\Audioware\\Service.reds`),
+        path.join(`r6\\scripts\\Audioware\\Settings.reds`),
+        path.join(`r6\\scripts\\Audioware\\System.reds`),
+        path.join(`r6\\scripts\\Audioware\\Tween.reds`),
+        path.join(`r6\\scripts\\Audioware\\Utils.reds`)
+      ],
+      outInstructions: [
+        createdDirectory(`r6\\audioware\\`), // This is a special case
+        copiedToSamePath(`red4ext\\plugins\\Audioware\\audioware.dll`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Codeware.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Config.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Ext.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Hooks.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Natives.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Preset.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Service.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Settings.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\System.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Tween.reds`),
+        copiedToSamePath(`r6\\scripts\\Audioware\\Utils.reds`)
+       
+      ],
+    },
+    coreAudiowareInstallCanonWithRandomStuff: {
+      expectedInstallerType: InstallerType.CoreAudioware,
+      inFiles: [
+        path.join(`red4ext\\`),
+        path.join(`red4ext\\plugins\\`),
+        path.join(`red4ext\\plugins\\iguesswhatever.archive`),
+        path.join(`red4ext\\plugins\\Audioware\\`),
+        path.join(`red4ext\\plugins\\Audioware\\audioware.dll`),
+        path.join(`r6\\scripts\\Audioware\\Codeware.reds`),
+        path.join(`r6\\scripts\\Audioware\\Config.reds`),
+        path.join(`r6\\scripts\\Audioware\\Ext.reds`),
+        path.join(`r6\\scripts\\Audioware\\Hooks.reds`),
+        path.join(`r6\\scripts\\Audioware\\Natives.reds`),
+        path.join(`r6\\scripts\\Audioware\\Preset.reds`),
+        path.join(`r6\\scripts\\Audioware\\Service.reds`),
+        path.join(`r6\\scripts\\Audioware\\Settings.reds`),
+        path.join(`r6\\scripts\\Audioware\\System.reds`),
+        path.join(`r6\\scripts\\Audioware\\Tween.reds`),
+        path.join(`r6\\scripts\\Audioware\\Utils.reds`)
+      ],
+      outInstructions: [
+        createdDirectory(`r6\\tweaks\\`), // This is a special case
```

Comment:
```suggestion
        createdDirectory(`r6\\audioware\\`), // This is a special case
```

I think this is one of the failures.